ca/certificates.go

```diff
@@ -815,6 +817,11 @@ func CreateRootCA(rootCN string) (RootCA, error) {
 		return RootCA{}, err
 	}
 
+	key, err = pkcs8.ConvertECPrivateKeyPEM(key)
```

Comment:
It looks like it won't work if the CA key is encrypted using the deprecated `SWARM_ROOT_CA_PASSPHRASE` environment variable.